NF.csv

```diff
@@ -377,6 +377,10 @@ cNF98.4c,A primary cutaneous neurofibroma cell line; not broadly available. See
 Prss56Cre;R26mT,"Prss56Cre;R26mT is a transgenic non-human animal model for Neurofibromatosis type 1, wherein the Nf1 gene is specifically inactivated in BC cells and derivatives thereof.","","",FALSE,https://patents.google.com/patent/EP3420811A1/en,modelSystemName,,"","",Prss56Cre;R26mT,Class,Biosample,Model_System,Mouse_Model,"","",""
 Nf1-OPG,"","","",FALSE,"",modelSystemName,,"","",Nf1-OPG,Class,Biosample,Model_System,Mouse_Model,"","",""
 Nf1-OPG-Arg816,"","","",FALSE,"",modelSystemName,,"","",Nf1-OPG-Arg816,Class,Biosample,Model_System,Mouse_Model,"","",""
+Nf2-/- Schwann cell line,"A Nf2-/- mouse schwann cell line","","",FALSE,"",modelSystemName,,"","",Nf2-/-Schwann,Class,Biosample,Model_System,Cell_Line_Model,"","",""
```

Comment:
I have a couple of requests for these: 
- can you ask Roberta if there are more specific names for these cell lines? These are really vague
- can you ask Roberta if these cell lines are published in the literature or listed on cellosaurus? If so, we should default to the name listed on Cellosaurus. 

e.g. is this one https://web.expasy.org/cellosaurus/CVCL_7691 or https://web.expasy.org/cellosaurus/CVCL_JK03?